build.sbt

```diff
@@ -195,12 +217,16 @@ lazy val sbtScalaNative =
                    publishLocal in nscplugin,
                    publishLocal in nativelib,
                    publishLocal in javalib,
-                   publishLocal in scalalib)
+                   publishLocal in scalalib,
+                   publishLocal in llvmTools)
         .evaluated,
       publishLocal := publishLocal.dependsOn(publishLocal in tools).value
     )
     .dependsOn(tools)
 
+lazy val llvmTools =
+  project.in(file("llvm-tools")).settings(toolSettings)
```

Comment:
Separate sbt project for this seems to be an overkill. Why not have it as part of existing `tools` project?